.github/workflows/mr-admin-pr-deploy.yaml

```diff
@@ -0,0 +1,41 @@
+name: Deploy Pull Request Branch
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Misforstår jeg nå, eller kjører denne kun i main, tilsvarende den vanlige CI/CD-flyten?

Reply:
Jeg misforstår, kjører på `pull_request`, ikke `push` :+1:

Reply:
Men da er det et mulig overlapp med denne workflowen og CI/CD-workflowen siden begge kjører test + build.
Det er nok mulig å f.eks. legge til et pr-deployment step i CI/CD-workflowen som kun kjører på et visst event. 

En annen ting vi kan vurdere er om alle pull-requests trenger sin egen deployment, eller om vi innfører opt-in for deployment på pull-requests (feks ved at vi skriver en `deploy`-kommentar i PR'en, eller har en konvensjon på at branches med en viss prefix blir deployet). Hvis vi ønsker å merge inn noe kode som ikke skal testes live så er det litt kjedelig å måtte vente på en deployment man egentlig ikke trenger.

Tanker? Dette er også mulig å ta som del av en fremtidig optimalisering.

Reply:
Gode poenger! Signe og jeg vil se på del 1 i løpet av dagen.

Interessant tanke om del 2 også. Kanskje det finnes en mulighet i GH Actions for å manuelt skippe et steg og gå videre til neste? Det kunne kanskje vært en løsning de gangene vi trenger en hurtigfiks ut i miljø. Vi sjekker det ut.

Reply:
Denne vil vel strengt tatt trigge på alle endringer i repo'et, som gjør at en endring i mr-api må vente på at mr-admin-flate deployes :sweat_smile: 
Er nok lurt å legge på et `paths`-filter som gjør at denne kun trigger for riktig app.

Reply:
Det samme gjelder nok for comment-link og cleanup, hadde vært en fordel om de kun kjører for riktig app.

Reply:
Godt poeng!